WhaleSpotting/react-app/src/components/Profile.tsx

```diff
@@ -0,0 +1,51 @@
+import "../styles/Profile.scss";
+import "../styles/Home.scss";
+import React from "react";
+import { useState } from "react";
+import PageNav from "./PageNav";
+
+export function Profile(): JSX.Element {
+
+    const [feedToggle, setFeedToggle] = useState("Sightings");
+
+    return (
+        <div className="body">
+            <div className="banner">
+                <div className="outer-container">
+                    <div className="trophy-text-container">
+                        <h1 className="heading">UserName</h1>
+                        <p className="joined little-text"> Joined: June 2004</p>
+                        <div className="trophy-container">
+                            <p className="feature-text"> 15</p>
+                            <p className="reported little-text"> Reported <br></br> Sightings</p>
+                            <img className="trophy-image" src="https://picsum.photos/id/215/70" />
+                        </div>
+                        <img className="profile-image" src="https://picsum.photos/id/237/200" />
+                    </div>
+                    <div className="button-container">
+                        <a className="toggle" onClick={() => setFeedToggle("Sightings")}>Sightings</a>
```

Comment:
Could we make these buttons instead and make them similar to the other buttons? It may make sense to make a Buttons.scss file with `primary-button` and `secondary-button` classes

Reply:
Addressed. New buttons.scss file created with primary +secondary button styles.